src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpHttpMetricsExportClient.cs

```diff
@@ -0,0 +1,96 @@
+// <copyright file="OtlpHttpMetricsExportClient.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Runtime.CompilerServices;
+#if NET5_0_OR_GREATER
+using System.Threading;
+#endif
+using System.Threading.Tasks;
+using Google.Protobuf;
+using OtlpCollector = Opentelemetry.Proto.Collector.Metrics.V1;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient
+{
+    /// <summary>Class for sending OTLP metrics export request over HTTP.</summary>
+    internal sealed class OtlpHttpMetricsExportClient : BaseOtlpHttpExportClient<OtlpCollector.ExportMetricsServiceRequest>
```

Comment:
@alanwest I'm assuming we were just missing the client bits for this, but do you know if the collector actually supports metics over http?

Reply:
Correct, just missing the bits. The collector does support metrics over HTTP (logs too).

Reply:
#2522

Reply:
Cool thanks for confirming. I added a "Closes #2522" into the PR description. I can do logs too in a follow-up.